tests/Integration/Behaviour/Features/Scenario/Order/add_cart_rule_to_order.feature

```diff
@@ -0,0 +1,103 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s order --tags add-discounts-to-order
+@reset-database-before-feature
+Feature: Add discounts to order from Back Office (BO)
+  As a BO user
+  I need to be able to add discounts to existing orders from the BO
+
+  Background:
+    Given the current currency is "USD"
+    And country "US" is enabled
+    And there is customer "testCustomer" with email "pub@prestashop.com"
+    And customer "testCustomer" has address in "US" country
+    And customer "testCustomer" has an empty cart "dummy_cart"
+    And cart "dummy_cart" delivery and invoice address country for customer "testCustomer" is "US"
+    And cart dummy_cart delivery and invoice address for customer "testCustomer" is in "Florida" state of "US" country
+    And the module "dummy_payment" is installed
+    And I am logged in as "test@prestashop.com" employee
+    And I add 2 products "Mug The best is yet to come" to the cart "dummy_cart"
+    And I add order "bo_order1" with the following details:
+      | cart                | dummy_cart                 |
+      | message             | test                       |
+      | payment module name | dummy_payment              |
+      | status              | Awaiting bank wire payment |
+    And tax "Sales-taxes US-FL 6%" is applied to order bo_order1
+    And Order "bo_order1" has following prices:
```

Comment:
That's a great addition! This validate amounts along with currency
@sowbiba @Progi1984 You might be interested in this one in some of your behat tests